pyhive/tests/test_hive.py

```diff
@@ -204,14 +202,35 @@ def test_custom_transport(self):
         socket = thrift.transport.TSocket.TSocket('localhost', 10000)
         sasl_auth = 'PLAIN'
 
-        def sasl_factory():
+        spec_sasl = importlib.util.find_spec('sasl')
```

Comment:
qq, is importlib compatible with python 2?

Reply:
No. Shall i use `pkgutil.find_loader()` for compatibility with python 2?